Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/RepositoryItemFactory.cs

```diff
@@ -590,9 +590,16 @@ public ITextBoxFormatter CreateTextBoxFormatter(object Textblock)
         //}
 
        
+        public string GetALMConfig()
+        {
+            return WorkSpace.Instance.Solution.ALMConfigs.Where(x => x.DefaultAlm == true).FirstOrDefault().AlmType.ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)